geode-core/src/main/java/org/apache/geode/admin/internal/FinishBackupRequest.java

```diff
@@ -48,68 +47,92 @@
 /**
  * A request send from an admin VM to all of the peers to indicate that that should complete the
  * backup operation.
- *
- *
  */
 public class FinishBackupRequest extends CliLegacyMessage {
   private static final Logger logger = LogService.getLogger();
 
+  private final DM dm;
```

Comment:
Lets try to refactor the send logic and reply logic into another class whose job it is to create this message, send it and then wait for the reply.